### DISCUSSION TITLE
Access control within a set of DAGs

### DISCUSSION THREAD

**Discussion by youcefjd** [apache/airflow#28642] [General]
Hey, I am new to Airflow, and want to set up custom roles to allow specific teams to manage their DAGs and only their DAGs. The way I am thinking of doing it is to define the permissions at the TAG level. Tag being a team name. But I can only see that I can set permissions at the dag_id level, which makes my life a bit complicated; as I have to do it manually for each dag. I want to use tags as sort of AD groups. Is it possible?

Thanks

**Comment by potiuk** [comment]
Not yet. It's coming but not shortly. There is a proposal that is going to be published soon on devlist to implement more complete proposal on that, but this has not been publicly discussed (not even close to approve) yet.

**Reply by youcefjd** [reply]
Thanks for your reply. Is there any other mechanism I can leverage (besides tags) to perform control access operations over a group of DAGs, instead of doing it individually?

**Reply by potiuk** [reply]
All described in the docs: https://airflow.apache.org/docs/apache-airflow/stable/security/access-control.html#permissions

**Reply by youcefjd** [reply]
Right, but those are operations on the dag_id, which are unique. I am looking for ways to perform those operations on a group of DAGs. 

I am assuming I am stuck with dag_ids. Thank you for taking the time to reply.

**Reply by potiuk** [reply]
Yes. You are.
